doc/api/os.markdown

```diff
@@ -30,10 +30,26 @@ on OS X and `'Windows_NT'` on Windows.
 
 ## os.platform()
 
-Returns the operating system platform. Possible values are `'darwin'`,
-`'freebsd'`, `'linux'`, `'sunos'` or `'win32'`. Returns the value of
+Returns the operating system platform. Returns the value of
 `process.platform`.
 
+Its value is based on [`OS` constant of `gyp`](https://chromium.googlesource.com/external/gyp/+/HEAD/docs/InputFormatReference.md#Predefined-Variables)
+, but with exceptions in case of Mac OS X, Windows and Solaris.
+
+On Windows its value is always `"win32"`, on Mac OS X it's `"darwin"` and on
+Solaris it's `"sunos"`.
```

Comment:
> On Windows its value is always `"win32"`, on Mac OS X it's `"darwin"` and on
> Solaris it's `"sunos"`.

The windows exception shows that its always `win32` on all architectures of windows.
The Mac OS X and Solaris examples contrast the fact that the value of `OS` constant from `gyp` is not the same as the one returned from this function. 
| os | `gyp` | `os.platform` |
| --- | ------  | ---------------  |
|Windows | `win` | `win32 |
|Mac| `mac`| `darwin`|
|Solaris| `solaris` | `sunos`|